### ISSUE TITLE
Connect to previous is failing

### ISSUE THREAD

**Issue by SJLennon** [halcyon-tech/vscode-ibmi#284]
**Describe the bug**
Used 0.7.2 for the first time today and Connect to previous does nothing.

**To Reproduce**
F1, then select Connect to previous

**Expected behavior**
I expect it to put up my list of connections.

**Screenshots**
![image](https://user-images.githubusercontent.com/67484051/132893916-7f6a5749-704a-486c-a0db-d5c632759f4c.png)

**Environment (please complete the following information):**
 - Extension version [e.g. 0.7.2]
 - IBM i OS version [e.g. 7.4]

**Additional context**
- I can create a new connection and it connects, but I can't use Connect to previous.
- I deleted all connections and connection settings from settings.json but left in the code-for-ibm1.actions.  Then I created a new connection and added an IFS shortcut.  Tried F1 and Connect to previous again.  It signed me off and showed the (new) connections listing in the sidebar. Tried F1 and Connect to previous again. .Still no luck.  
 Created another new connection. Tried F1 and Connect to previous again.  Still no luck. 
- If I add a new connection with an existing name, then it connects and honors the configuration settings I originally has for this configuration.

Settings.json:
```
{
    "telemetry.enableTelemetry": false,
    "editor.fontLigatures": false,
    "editor.renderWhitespace": "none",
    "editor.cursorStyle": "block",
    "overtype.labelInsertMode": "Ins",
    "overtype.labelOvertypeMode": "Ovr",
    "[Log]": {
        "editor.fontSize": 11
    },
    "terminal.integrated.shell.windows": "C:\\WINDOWS\\System32\\cmd.exe",
    "window.zoomLevel": 1,
    "update.mode": "start",
    "[javascript]": {
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
    "[html]": {
        "editor.defaultFormatter": "vscode.html-language-features"
    },
    "workbench.activityBar.iconClickBehavior": "focus",
    "workbench.list.openMode": "doubleClick",
    "npm.exclude": "",

    "workbench.editorAssociations": {
        "*.ipynb": "jupyter.notebook.ipynb"
    },
    "editor.fontFamily": " Lucida Console, Inconsolata, Consolas,  'Courier New', monospace",
    "workbench.colorTheme": "Visual Studio Dark",
    "git.autofetch": true,
    "editor.minimap.enabled": false,
    "code-for-ibmi.rpgleContentAssistEnabled": true,
    "code-for-ibmi.rpgleIndentationEnabled": true,
    "code-for-ibmi.logCompileOutput": true,
    "code-for-ibmi.actions": [
        {
            "type": "member",
            "extensions": [
                "RPGLE",
                "RPG"
            ],
            "name": "Create Bound RPG Program (CRTBNDRPG)",
            "command": "CRTBNDRPG PGM(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"
        },
        {
            "type": "member",
            "extensions": [
                "SQLRPGLE"
            ],
            "name": "Create SQL ILE RPG Program (CRTSQLRPGI)",
            "command": "?CRTSQLRPGI OBJ(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) CLOSQLCSR(*ENDMOD) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"
        },
        {
            "type": "member",
            "extensions": [
                "RPGLE",
                "RPG"
            ],
            "name": "Create RPG Module (CRTRPGMOD)",
            "command": "?CRTRPGMOD MODULE(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"
        },
        {
            "type": "member",
            "extensions": [
                "CMD"
            ],
            "name": "Create Command (CRTCMD)",
            "command": "?CRTCMD CMD(&OPENLIB/&OPENMBR) PGM(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) ALLOW(*ALL) CURLIB(*NOCHG) PRDLIB(*NOCHG)"
        },
        {
            "type": "member",
            "extensions": [
                "CMD"
            ],
            "name": "Create Command (Allow Return Variables - CRTCMD)",
            "command": "?CRTCMD CMD(&OPENLIB/&OPENMBR) PGM(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) ALLOW(ALLOW(*BPGM *IPGM *BMOD *IMOD) ) CURLIB(*NOCHG) PRDLIB(*NOCHG)"
        },
        {
            "type": "member",
            "extensions": [
                "pf",
                "lf"
            ],
            "name": "Delete File (DLTF)",
            "command": "?DLTF FILE(&OPENLIB/&OPENMBR)"
        },
        {
            "type": "member",
            "extensions": [
                "dspf"
            ],
            "name": "Create Display File (CRTDSPF)",
            "command": "?CRTDSPF FILE(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) SRCMBR(&OPENMBR) RSTDSP(*NO) REPLACE(*YES)"
        },
        {
            "type": "member",
            "extensions": [
                "pf"
            ],
            "name": "Create Physical File (CRTPF)",
            "command": "?CRTPF FILE(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) SRCMBR(&OPENMBR) SIZE(*NOMAX) MAXMBRS(1)"
        },
        {
            "type": "member",
            "extensions": [
                "lf"
            ],
            "name": "Create :Logical File (CRTLF)",
            "command": "?CRTLF FILE(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF)"
        },
        {
            "type": "member",
            "extensions": [
                "CLP",
                "CLLE"
            ],
            "name": "Create Bound CL Program (CRTBNDCL)",
            "command": "?CRTBNDCL PGM(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) OPTION(*EVENTF) DBGVIEW(*SOURCE)"
        },
        {
            "type": "member",
            "extensions": [
                "CLP",
                "CLLE"
            ],
            "name": "Create CL Program (CRTCLP)",
            "command": "?CRTCLPGM PGM(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) OUTPUT(*PRINT) REPLACE(*YES)"
        },
        {
            "type": "member",
            "extensions": [
                "PNLGRP"
            ],
            "name": "Create Panel Group (CRTPNLGRP)",
            "command": "?CRTPNLGRP PNLGRP(&OPENLIB/&OPENMBR) SRCFILE(&OPENLIB/&OPENSPF) SRCMBR(&OPENMBR) OPTION(*EVENTF)"
        },
        {
            "type": "member",
            "extensions": [
                "GLOBAL"
            ],
            "name": "Create Program (CRTPGM)",
            "command": "?CRTPGM PGM(&OPENLIB/&OPENMBR) MODULE(*PGM) ENTMOD(*FIRST) BNDSRVPGM(*NONE) BNDDIR(*NONE) ACTGRP(*ENTMOD) TGTRLS(*CURRENT)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "rpgle"
            ],
            "name": "Create Bound RPG Program (CRTBNDRPG) with inputs",
            "command": "CRTBNDRPG PGM(${buildlib|Build library|&BUILDLIB}/${objectname|Object Name|&NAME}) SRCSTMF('${sourcePath|Source path|&FULLPATH}') OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "RPGLE",
                "RPG"
            ],
            "name": "Create Bound RPG Program (CRTBNDRPG)",
            "command": "CRTBNDRPG PGM(&BUILDLIB/&NAME) SRCSTMF('&FULLPATH') OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "SQLRPGLE"
            ],
            "name": "Create SQL ILE RPG Program (CRTSQLRPGI)",
            "command": "?CRTSQLRPGI OBJ(&BUILDLIB/&NAME) SRCSTMF('&FULLPATH') CLOSQLCSR(*ENDMOD) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "RPGLE",
                "RPG"
            ],
            "name": "Create RPG Module (CRTRPGMOD)",
            "command": "?CRTRPGMOD MODULE(&BUILDLIB/&NAME) SRCSTMF('&FULLPATH') OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "CLP",
                "CLLE"
            ],
            "name": "Create Bound CL Program (CRTBNDCL)",
            "command": "?CRTBNDCL PGM(&BUILDLIB/&NAME) SRCSTMF('&FULLPATH') OPTION(*EVENTF) DBGVIEW(*SOURCE)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "cmd"
            ],
            "name": "Create Command (CRTCMD)",
            "command": "?CRTCMD CMD(&BUILDLIB/&NAME) PGM(&BUILDLIB/&NAME) SRCSTMF('&FULLPATH') OPTION(*EVENTF)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "cmd"
            ],
            "name": "Create Command (Allow Return Variables - CRTCMD)",
            "command": "?CRTCMD CMD(&BUILDLIB/&NAME) PGM(&BUILDLIB/&NAME) SRCSTMF('&FULLPATH') ALLOW(*BPGM *IPGM *BMOD *IMOD *BREXX *IREXX) OPTION(*EVENTF)"
        },
        {
            "type": "streamfile",
            "extensions": [
                "GLOBAL"
            ],
            "name": "Set CCSID to 1252",
            "environment": "pase",
            "command": "setccsid 1252 &FULLPATH"
        },
        {
            "type": "object",
            "extensions": [
                "GLOBAL"
            ],
            "name": "Delete object",
            "command": "?DLTOBJ OBJ(&LIBRARY/&NAME) OBJTYPE(*&TYPE)"
        },
        {
            "type": "object",
            "extensions": [
                "pgm"
            ],
            "name": "Call program",
            "command": "?CALL &LIBRARY/&NAME"
        }
    ],

    "editor.mouseWheelZoom": true,
    "editor.bracketPairColorization.enabled": true,
    "editor.fontSize": 12,
    "code-for-ibmi.connectionSettings": [
        {
            "name": "ipubtest",
            "sourceFileList": [
                "QSYSINC/H"
            ],
            "objectBrowserList": [
                "LENNONS1",
                "LENNONS2",
                "QTEMP",
                "QGPL"
            ],
            "databaseBrowserList": [
                "LENNONS1",
                "LENNONS2",
                "QTEMP",
                "QGPL"
            ],
            "libraryList": [
                "LENNONS1",
                "LENNONS2",
                "QTEMP",
                "QGPL"
            ],
            "libraryListProfiles": [],
            "ifsShortcuts": [
                "/home/LENNONS",
                "/home/lennons/github"
            ],
            "homeDirectory": "/home/LENNONS",
            "tempLibrary": "LENNONS1",
            "currentLibrary": "LENNONS1",
            "sourceFileCCSID": "*FILE",
            "enableSQL": true,
            "hideCompileErrors": [],
            "enableSourceDates": false,
            "sourceDateLocation": "none",
            "clContentAssistEnabled": false
        },
        {
            "name": "ipubxx",
            "sourceFileList": [
                "QSYSINC/H"
            ],
            "objectBrowserList": [
                "LENNONS1",
                "LENNONS2",
                "QTEMP",
                "QGPL"
            ],
            "databaseBrowserList": [
                "LENNONS1",
                "LENNONS2",
                "QTEMP",
                "QGPL"
            ],
            "libraryList": [
                "LENNONS1",
                "LENNONS2",
                "QTEMP",
                "QGPL"
            ],
            "libraryListProfiles": [],
            "ifsShortcuts": [
                "/home/LENNONS"
            ],
            "homeDirectory": "/home/LENNONS",
            "tempLibrary": "LENNONS1",
            "currentLibrary": "LENNONS1",
            "sourceFileCCSID": "*FILE",
            "enableSQL": true,
            "hideCompileErrors": [],
            "enableSourceDates": false,
            "sourceDateLocation": "none",
            "clContentAssistEnabled": false
        }
    ],
    "code-for-ibmi.connections": [
        {
            "name": "ipubtest",
            "host": "ipublic.online",
            "port": 2222,
            "username": "lennons",
            "privateKey": null
        },
        {
            "name": "ipubxx",
            "host": "ipublic.online",
            "port": 2222,
            "username": "lennons",
            "privateKey": null
        }
    ],
}
```

**Comment by worksofliam** [comment]
@SJLennon Thanks for the issue.

Connect to previous is now useless when running as a command from the user. I should make an issue to add a notice since the command is now used by the new browser.

To 'connect to previous' you can just select the connection from the list in the panel and it will do the same.

**Comment by worksofliam** [comment]
Created #285

**Comment by SJLennon** [comment]
Duh!  Didn't think to double click.  Did try right click though.

**Comment by worksofliam** [comment]
I will close this for now. Thanks for raising your concerns.. I will make sure to make improvements from it!

**Comment by SJLennon** [comment]
I've been tempted to make some changes to the documentation, but I'm not sure I can keep up with you.

**Comment by worksofliam** [comment]
@SJLennon yes, this is a problem I was thinking about last night.

I really like our quick deployment method but keeping the docs up to date is tough.

The important part, which I plan to stay on top of, is the walkthroughs. They are very important for newbies.

**Comment by SJLennon** [comment]
I'm not totally up to speed on how walkthroughs are supposed to work.  Right now I see them in Welcome, like below. But...
1. I see only two.  I know there are more because I edited at least three, and I think I saw three at one point.
2. Should they be appearing anywhere else?  
3. Have I done something to hide them?
![image](https://user-images.githubusercontent.com/67484051/132929038-e555a263-7d74-46cf-8d48-6d2e6389bebf.png)
Just curious...
